src/diffusers/models/attention.py

```diff
@@ -290,6 +290,33 @@ def __init__(
         self.rescale_output_factor = rescale_output_factor
         self.proj_attn = nn.Linear(channels, channels, 1)
 
+        self._use_memory_efficient_attention_xformers = False
+
+    def _set_use_memory_efficient_attention_xformers(self, use_memory_efficient_attention_xformers: bool):
```

Comment:
Let's make this public. And thanks to #1493, it's the only place where we need to add this method!

```suggestion
    def set_use_memory_efficient_attention_xformers(self, use_memory_efficient_attention_xformers: bool):
```

Reply:
Thanks, done!